apps/showcase/src/app/app.component.html

```diff
@@ -27,7 +29,7 @@
       <div class="col-12 col-md-8">
       </div>
       <div class="col-12 col-md-4">
-        <h5>Links</h5>
+        <h2 class="h5">Links</h2>
```

Comment:
Do we know why this is better for the accessibility score ? Do we have previously an h1 and it expects an h2 ?

Reply:
the headings needs to be in the right order for better navigation and integration with screen-readers
here is the link https://dequeuniversity.com/rules/axe/4.8/heading-order proposed by lighthouse to explain it